Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -55,6 +59,15 @@ final class UIViewControllerExtensionsTests: XCTestCase {
         XCTAssertFalse(viewController.notificationFired)
     }
 
+    #if !targetEnvironment(macCatalyst)
+    
+    func testInstantiate() {
+        let myViewController = MyViewController.instantiate(from: "TestStoryboard")
+        XCTAssertNotNil(myViewController.testLabel)
```

Comment:
You will need to call `myViewController.loadViewIfNeeded()` before this or it will fail.